### ISSUE TITLE
DOCS: update to include handling other project ID updates and conflict resolution process

### ISSUE THREAD

**Issue by kurtseifried** [distributedweaknessfiling/dwf-workflow#18]
DOCS: update to include handling other project ID updates and conflict resolution process

TL;DR: If we assign a DWF vulnerability Identifier that is CVE compatible, and MITRE or a CNA assigns one and it comes to our attention we will 

1) mark ours as DISPUTED
2) Investigate to make sure they are the same thing
3) If they are the exact same thing we'll simply mark ours as a DUPLICATE of the MITRE or CNA one for simplicity  as it's difficult to update the MITRE CVE database and we can do it faster
4) If they overlap we may need to do some SPLIT / MERGE work and update our identifier
5) If they don't overlap then we move from DISPUTED back to the original state, leave notes/etc explaining what happened in the JSON.

Does the above make sense? I don't think it has any major flaw. Thoughts/comments?

**Comment by joshbressers** [comment]
This sounds good to me

**Comment by kurtseifried** [comment]
One addendum: we don't mark it as a duplicate until it's published in the NVD database.
